docs/nodejs.md

```diff
@@ -0,0 +1,161 @@
+<!--
+title: Node.js SDK
+menuText: Node.js SDK
+description: 
+menuOrder: 4
+-->
+
+# Node.js SDK
+
+Serverless Console, when Tracing is enabled on an AWS Lambda function, will hook
+into the AWS Lambda runtime environment and automatically report Traces and
+Spans. To capture handled errors, warnings, and to set custom tags, the SDK
+library must be added and instrumented in your AWS Lambda function handler.
+
+## Key terms
+
+- A **Captured Error** is one instance of an `Error` object in Node.js that is
+sent to Serverless Console. It can be viewed in Dev Mode or the Trace Explorer
+Details.
+- A **Captured Warning** is one instance of a string in Node.js that is sent to
+Serverless Console, much like a Captured Error.
+- A **Tag** is a key/value-pair that can be set on the Trace or an individual
+Captured Error or Captured Warning, and sent to Serverless Console. Tags can be
+viewed on the Trace Explorer Details and Dev Mode.
+
+## Installation
+
+**Install the package with:**
+
+```javascript
+npm install @serverless/sdk --save
+# or
+yarn add @serverless/sdk
+```
+
+**Enable Tracing, Logging, and Dev Mode**
+
+The SDK will merely generate the necessary Tags, Captured Errors, and Captured
+Warnings; however, Tracing, Logging, and Dev Mode must be enabled on your org on
+Serverless Console in order for the data to be ingested.
+
+[Enable Tracing, Logging, and Dev Mode](/console/docs/integrations/enable-monitoring-features)
+
+## Usage
+
+The package does not require any configuration as the credentials are
+automatically set on the AWS Lambda function environment variables when Tracing
+is enabled in Serverless Console.
+
+To use the Serverless SDK you must require the `@serverless/sdk` method in your
+AWS Lambda function handler.
+
+```javascript
+const serverlessSdk = require("@serverless/sdk");
+```
```

Comment:
It's worth here to put some light on difference between `@serverless/sdk` and `@serverless/aws-lambda-sdk`

In Lambda environment importing either will export exactly same SDK. However when working locally, it's only `@serverless/aws-lambda-sdk` which will export AWS Lambda specific instrumentation methods properties, which are documented here: https://github.com/serverless/console/blob/main/node/packages/aws-lambda-sdk/docs/sdk.md (also in VS Code the intellisense for those methods will only be exposed if `@serverless/aws-lambda-sdk` is imported)

So possibly best approach is to rely on `@serverless/sdk` in environment agnostic logic, while for AWS Lambda specific code it's safer to rely on `@serverless/aws-lambda-sdk`

Reply:
I added content on using the manual instrumentation; however, we couldn't get the fix in for the manual instrumentation, so I've removed it for now. Next week, when we release the fix I'll publish the updated docs with reference to `@serverless/aws-lambda-sdk` and the manual instrumentation. The proposed changes are in the Linear issue.